doc/nrf/ug_nrf9160.rst

```diff
@@ -199,8 +201,66 @@ Likewise, in Step 7, choose the :file:`.hex` file for the application you are pr
 .. note::
    When you update the application firmware on an nRF9160 DK, you must also update the modem firmware as described in :ref:`nrf9160_gs_updating_fw_modem`.
 
-See :ref:`gs_programming` for more information on building and programming samples on nRF9160 DK.
+.. _build_pgm_nrf9160_vsc:
+
+Building and programming using |VSC|
+====================================
+
+|vsc_extension_instructions|
+
+Complete the following steps after installing the |nRFVSC|:
+
+.. |sample_path_vsc| replace:: :file:`samples/nrf9160/aws_iot`
```

Comment:
Why `aws_iot` for Visual Studio Code, and not Asset Tracker v2 that is used for command line?

Reply:
The AWS IoT sample requires the users to set up an AWS account and set up IoT Hub to make use of the sample on the device, so I'm not sure it's the best example to start with. http://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/samples/nrf9160/aws_iot/README.html#setup.